CMakeLists.txt

```diff
@@ -37,9 +37,14 @@ set_target_properties(essim PROPERTIES CXX_CLANG_TIDY "${CLANG_TIDY_COMMAND}")
 target_include_directories(essim PRIVATE
   ${CMAKE_SOURCE_DIR}
   )
-set_property(SOURCE "essim/src/ssim_internal.x86.sse41.c" "essim/src/ssim_internal.x86.ssse3.c"  PROPERTY COMPILE_FLAGS "-mssse3 -msse4.1")
-set_property(SOURCE "essim/src/ssim_internal.x86.avx2.c" PROPERTY COMPILE_FLAGS "-mavx2")
-set_target_properties(essim PROPERTIES COMPILE_FLAGS "-fexceptions -fpermissive -Wall -Wpedantic -mxsave")
+
+if(${CMAKE_SYSTEM_PROCESSOR} STREQUAL "aarch64")
```

Comment:
on my laptop this reports `arm64` rather than `aarc64`, so we should use something like

```
if(CMAKE_SYSTEM_PROCESSOR MATCHES "aarch64|arm64")
```

to check for either case